docs/source/playbooks.rst

```diff
@@ -228,6 +228,28 @@ for Python and ``ZOAU`` have been added and used through the configuration.
      _TAG_REDIR_OUT: "txt"
      LANG: "C"
 
+.. note::
+
+   Currently, IBM Open Enterprise Python for z/OS is the supported and
+   recommended Python distribution for use on z/OS with Ansible and ZOAU. If
+   Rocket Python is the only available python on the target, please review the
+   suggested environment variables for use with Rocket Python.
+
+file:///Users/ddimatos/git/github/ibm_zos_core/docs/build/html/playbooks.html#group-vars
+
+.. code-block:: yaml
+
+   PYZ: "/usr/lpp/rsusr/python36"
+   ZOAU: "/usr/lpp/IBM/zoautil"
+
+   environment_vars:
+     ZOAU_ROOT: "{{ ZOAU }}"
+     ZOAU_HOME: "{{ ZOAU }}"
+     PYTHONPATH: "{{ ZOAU }}/lib:{{ PYZ }}:/lib:/usr/lib"
+     _BPXK_AUTOCVT: "ON"
+     PATH: "{{ ZOAU }}/bin:/bin:/var/bin:{{ PYZ }}/bin"
+     LIBPATH: "{{ ZOAU }}/lib:{{ PYZ }}/lib:/lib:/usr/lib"
```

Comment:
Do we want to add `.` to the end here?

Reply:
Fixed, good catch, funny I caught this the other day for you :)

Reply:
I think they were different files, but your catch the other day led me to check this one too 😄